osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -60,8 +61,20 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
             BeatmapSetInfo beatmapSetInfo = beatmap.BeatmapSetInfo;
             BeatmapInfo beatmapInfo = beatmap.BeatmapInfo;
 
+            double bpmPreview = 60000 / beatmap.Beatmap.BeatLengthAt(beatmap.Beatmap.Metadata.PreviewTime);

```

Comment:
You don't need this. Just set bpmMax to double.MinValue and bpmMin to double.MaxValue initially.